crowdin.yml

```diff
@@ -1,3 +1,3 @@
 files:
-  - source: /packages/smooth_app/lib/l10n/intl_en.arb
-    translation: /packages/smooth_app/lib/l10n/intl_%locale_with_underscore%.arb
+  - source: /packages/smooth_app/lib/l10n/app_en.arb
+    translation: /packages/smooth_app/lib/l10n/app_%two_letters_code%.arb
```

Comment:
can we have proper local support for
pt_PT, pt_BR, zh_CN, zh_TW…

Reply:
Flutter needs locales with just the language code, we can add "pt_BR", but "pt" has to be there:

``````
flutter pub get
Arb file for a fallback, fr, does not exist, even though 
the following locale(s) exist: [fr_FR]. 
When locales specify a script code or country code, a 
base locale (without the script code or country code) should 
exist as the fallback. Please create a {fileName}_fr.arb 
file.
Generating synthetic localizations package has failed.

Reply:
Tested yesterday: https://github.com/openfoodfacts/smooth-app/issues/43#issuecomment-751386730